db/model/model.js

```diff
@@ -165,3 +165,17 @@ exports.orderArticles = (sort_by, order_by) => {
     return rows;
   });
 };
+
+exports.filterByTopics = (topics) => {
+  console.log(topics);
+  const validInputs = ["mitch", "cats"];
+  if (!validInputs.includes(topics)) {
+    return Promise.reject({ status: 400, msg: "bad request" });
+  }
```

Comment:
This isn't necessary as below this code you've already protected the query from SQL injection with the dollar syntax